Api.Flickr/Models/FlickrPhotosetsGetPhotosResponseStatus.cs

```diff
@@ -0,0 +1,15 @@
+namespace Api.Flickr.Models
+{
+    /// <summary>
+    /// Statuses of response for photosets.getPhotos method
+    /// </summary>
+    public enum FlickrPhotosetsGetPhotosResponseStatus
+    {
+        Okay,
+        PhotosetNotFound,
+        UserNotFound,
+        InvalidApiKey,
```

Comment:
We still can send custom messages in Bad Request. I think that enum is "bicycle".

Reply:
Flickr API returns 200 OK http status code for errors including codes and messages. You would prefer to remove this enum and leave only messages?

Reply:
Or we can raise default error messages.